src/app/pages/OnDemandTvPage/index.test.jsx

```diff
@@ -24,13 +24,12 @@ const toggles = {
   },
 };
 
-const Page = ({ pageData, service, isAmp = false }) => (
+const Page = ({ pageData, service = false }) => (
```

Comment:
The `false` default belonged with the `isAmp` param and needs to be removed

```suggestion
const Page = ({ pageData, service }) => (
```